src/components/displayQuote.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import Loader from './Loader';
+
+const DisplayQuote = () => {
+  const [quote, setQuote] = useState(null);
+  const [error, setError] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchQuote = async () => {
+      try {
+        const API_KEY = 'https://api.api-ninjas.com/v1/quotes?category=funny';
+        const response = await fetch(
+          API_KEY,
+          {
+            headers: {
+              'X-Api-Key': 'RSEoeXesveyZYofHdotTdw==uMk2w3sAMDYOZZPA',
```

Comment:
- [ ] [OPTIONAL] Here, your API key should always be protected at all costs. I know it isn't a hard requirement but kindly consider putting it in an environment variable to keep it safe. 
#### Note:
If you need a guide on how to use React environment variables, check this [article](https://www.architect.io/blog/2022-08-16/react-environment-variables-developers-guide/)